docs/index.md

```diff
@@ -1 +0,0 @@
-../README.md
```

Comment:
We have to create a dedicated `index.md` file to have a proper link to the website page and not to Github.

Reply:
Also, this file is basically a copy-paste of the current `README.md` besides the 'documentation' paragraph. It's odd a little to have this paragraph when we have the bar with those links on every page.